src/Core/Command/Repo/SetOptionsCommand.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace DigipolisGent\Github\Core\Command\Repo;
+
+use DigipolisGent\Github\Core\Handler\RepositoriesHandler;
+use DigipolisGent\Github\Core\Log\ConsoleLogger;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Set some options for the repositories of a team.
+ *
+ * @package DigipolisGent\Github\Command\Repo
+ */
+class SetOptionsCommand extends ListCommand
+{
+    /**
+     * @inheritdoc
+     */
+    protected function configure()
+    {
+        parent::configure();
+
+        $this
+            ->setName('repo:set-options')
+            ->setDescription('Set options for repositories.')
+            ->setHelp('Set some options for the specified repositories.');
+
+        $this->addOption(
+            'has-issues',
+            'i',
+            InputOption::VALUE_OPTIONAL,
+            'Enable or disable issues.'
+        );
+
+        $this->addOption(
+            'has-wiki',
+            'w',
+            InputOption::VALUE_OPTIONAL,
+            'Enable or disable projects.'
+        );
+
+        $this->addOption(
+            'has-projects',
+            'p',
+            InputOption::VALUE_OPTIONAL,
+            'Enable or disable projects.'
+        );
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+        $logger = new ConsoleLogger($io);
+
+        // Get the specified options.
+        $options = [];
+
+        if ($this->isOptionSpecified($input, 'has-issues')) {
+            $options['has_issues'] = $this->getBoolOption($input, 'has-issues');
+        }
+
+        if ($this->isOptionSpecified($input, 'has-wiki')) {
+            $options['has_wiki'] =  $this->getBoolOption($input, 'has-wiki');
+        }
+
+        if ($this->isOptionSpecified($input, 'has-projects')) {
+            $options['has_projects'] =  $this->getBoolOption($input, 'has-projects');
+        }
+
+        if (!$options) {
```

Comment:
If there are no options specified, the defaults (all false, see above) should be followed.